pkg/util/devices.go

```diff
@@ -174,3 +174,13 @@ func isDiskOrDiskController(dev vimTypes.BaseVirtualDevice) bool {
 		return false
 	}
 }
+
+// isNonRDMDisk returns true for all virtual disk devices excluding disks with a raw device mapping backing and other device types.
```

Comment:
what are included in  `and other device types`?

Reply:
agreed. That comment is a little confusing. Fixed it.